shared/templates/file_groupowner/oval.template

```diff
@@ -13,59 +13,59 @@
    {{% endif %}}
     </criteria>
   </definition>
-
+  
+  {{%- set GROUPS=GID_OR_NAME.split("|") %}}
+  {{%- if GROUP_REPRESENTED_WITH_GID %}}
+  <local_variable id="var_file_groupowner{{{ FILEID }}}_{{{ GID_OR_NAME }}}_gid" comment="Set the gid to {{{ GID_OR_NAME }}}" datatype="int" version="1">
+    <literal_component datatype="int">{{{ GID_OR_NAME }}}</literal_component>
+  </local_variable>
+  {{%- else %}}
+  {{% for grp in GROUPS %}}
+    <ind:textfilecontent54_object id="object_file_groupowner{{{ FILEID }}}_{{{grp}}}_gid" version="1" comment="gid of the {{{grp}}} group">
```

Comment:
Does it mean that it doesn't support multiple groups expressed numerically, ie. if I want multiple groups I always need to specify them by their name? I'm fine with this limitation, but please ensure that nobody could use that in their rules. You can raise an exception in the template.py if somebody provides multiple groups in form of GID numbers.

Reply:
Documented and added check to template.py